svc/pkg/mm-config/ops/version-get/src/lib.rs

```diff
@@ -53,52 +53,43 @@ async fn handle(
 		.map(common::Uuid::as_uuid)
 		.collect::<Vec<_>>();
 
-	// TODO: There's a bug with this that returns the lobby groups for the wrong
-	// version, can't figure this out
```

Comment:
@NathanFlurry lmk if you have more info about this issue, the code looks fine to me